src/lib.ts

```diff
@@ -1,17 +1,24 @@
 import * as core from '@actions/core'
 import * as exec from '@actions/exec'
+import { ExecOptions } from '@actions/exec'
 import * as io from '@actions/io'
 
+async function execBC(arg: string): Promise<void> {
+  const env = { ...process.env } as ExecOptions['env']
```

Comment:
There is an `import * as exec from '@actions/exec'` already, curious if we can just use the common import instead of a specific `ExecOptions` import here? Then there is no need to change imports, makes the diff even smaller

```suggestion
  const env = { ...process.env } as exec.ExecOptions['env']
```